modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/PageToolbar.js

```diff
@@ -50,67 +128,81 @@ function EditTitleModal({
 		}
 	};
 
+	const _getTitleLabel = () => (
+		<>
+			{Liferay.Language.get('title')}
+
+			{selectedLocale.label === defaultLocaleBCP47 && (
+				<ClayIcon
+					className="ml-1 reference-mark"
+					focusable="false"
+					role="presentation"
+					symbol="asterisk"
+				/>
+			)}
+		</>
+	);
+
 	return (
 		<ClayModal
-			className="entry-edit-title-modal"
+			className="sxp-blueprint-edit-title-modal"
 			observer={observer}
 			size="md"
 		>
 			<ClayForm onSubmit={_handleSubmit}>
 				<ClayModal.Body>
-					<ClayForm.Group className={hasError ? 'has-error' : ''}>
-						<label htmlFor="title">
-							{Liferay.Language.get('title')}
-
-							<ClayIcon
-								className="ml-1 reference-mark"
-								focusable="false"
-								role="presentation"
-								symbol="asterisk"
-							/>
-						</label>
-
-						<ClayInput
+					<div
+						className={getCN('edit-title', {'has-error': hasError})}
+					>
+						<ClayLocalizedInput
 							autoFocus={modalFieldFocus === 'title'}
 							id="title"
-							onBlur={({currentTarget}) => {
-								setHasError(!currentTarget.value);
-							}}
-							onChange={({target: {value}}) => setTitle(value)}
+							label={_getTitleLabel()}
+							locales={localesForSelector}
+							onBlur={_handleBlur}
+							onSelectedLocaleChange={_handleChangeLocale(
```

Comment:
Match handler name with prop. In this case it should be `_handleSelectedLocaleChange`.

Let's also follow the pattern of `_handle*Change`, where specific names go after `_handle` so the handler functions are grouped together by component (if there are multiple handlers like `_handle*Blur`, `_handle*Focus`).

Reply:
Added, thanks! `_handle*Change` got it 👍